modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -127,10 +127,9 @@ public void start(NHttpClientConnection conn) throws IOException, HttpException
         String contentLengthHeader = null;
         LinkedHashSet<String> httpContentLengthHeader = headers.get(HTTP.CONTENT_LEN);
          
-	
-	    if(httpContentLengthHeader != null && httpContentLengthHeader.iterator().hasNext()) {   
-	    	contentLengthHeader = httpContentLengthHeader.iterator().next();
-	    }
+        if(httpContentLengthHeader != null && httpContentLengthHeader.iterator().hasNext()) {
```

Comment:
Format the code

Reply:
what does it mean?

Reply:
This thould be like this:
if (httpContentLengthHeader != null && httpContentLengthHeader.iterator().hasNext())

Reply:
I missed this part. I added additional commit.